testgrid/config.yaml

```diff
@@ -2172,31 +2172,31 @@ test_groups:
   num_failures_to_alert: 1
 # Istio daily releases
 - name: daily-e2e-rbac-no_auth
-  gcs_prefix: istio-prow/daily-e2e-rbac-no_auth
+  gcs_prefix: istio-prow/pull/istio-releases_daily-release/*/daily-e2e-rbac-no_auth
```

Comment:
usually we have a path like `kubernetes-jenkins/pr-logs/directory/pull-kops-bazel-build` which seems to work https://k8s-testgrid.appspot.com/presubmits-kops#e2e

Reply:
Aha the txt files in directory/ store the original path with the pull number. Thanks Ben! Please take a look on my latest change